Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.ts

```diff
@@ -14,10 +18,48 @@ import { SetDisplayedNotificationVersion } from 'src/app/shared/states';
 })
 export class DisplayNotificationDialog implements OnInit, OnDestroy {
   public static version: number = 1;
-  constructor(public dialogRef: MatDialogRef<DisplayNotificationDialog>, store: Store) {
+  constructor(
+    public dialogRef: MatDialogRef<DisplayNotificationDialog>,
+    store: Store,
+    private router: Router,
+    private snackBar: MatSnackBar,
+    private i18n: I18n,
+    private sanitizer: DomSanitizer
+  ) {
     store.dispatch(new SetDisplayedNotificationVersion(DisplayNotificationDialog.version));
   }
 
+  public openSnackBar() {
+    const url = this.sanitizer.bypassSecurityTrustHtml(
+      '<a href="' + window.location.href + '?skip_dialog=true">Link</a>'
+    );
+    const snackBarRef = this.snackBar
+      .open(
+        this.i18n({
+          meaning: 'Display-New-Url-Advice',
```

Comment:
You dont need to set `-` Meaning is just text to explain more or give context to the translator (e.g. by referring the component name)